jbmc/regression/jbmc/symex_can_track_when_clinits_have_definitely_been_run/test.desc

```diff
@@ -0,0 +1,16 @@
+CORE
+Test.class
+--function Test.test --show-vcc
+^EXIT=0$
+^SIGNAL=0$
+--
+java::ClassWithStaticInit::clinit_already_run#[1-9][0-9]
+^warning: ignoring
+--
+After the second call to ClassWithStaticInit.getStaticValue(), symex should have
```

Comment:
Good point Peter. Actually, the number of generations per call could varies depending on whether `apply_condition` thinks it can set a value to be constant on one or both of the branches, and also future changes. I think it's safest to just say that we expect at least one new generation per call if symex doesn't know that `clinit_already_run` is true. I have increased the number of static function calls to 20, to make absolutely sure that we'd go over if the feature isn't working properly.